src/sentry/tsdb/snuba.py

```diff
@@ -19,6 +20,15 @@
 )
 
 
+def to_project_filter_model_query(item, dataset, aggregate):
```

Comment:
the project_id is added to the groupby clause. Calling this project_filter seems counterintuitive.